Mail2Bug/Email/EWS/EWSIncomingMessage.cs

```diff
@@ -42,6 +47,16 @@ public EWSIncomingMessage(EmailMessage message)
         public string RawBody { get { return _message.Body.Text ?? string.Empty; } }
         public string PlainTextBody { get { return GetPlainTextBody(_message); } }
 
+        public string ConversationGuid
+        {
+            get
+            {
+                return _conversationId == null
+                           ? ConversationIndex.Substring(12, 32)
```

Comment:
Having two ways of getting it seems to me like a chance for introducing weird behavior in edge cases. If we expect this fallback option to work, why not use this always?

Regardless, can you please add documentation of why taking this substring satisfies this (and pointer to documentation of that, if such documentation exists)